spec/book_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'spec_helper'
+require_relative '../modules/label'
+require_relative '../modules/item'
+require_relative '../modules/book'
+
+describe Book do
+  before :each do
+    @book = Book.new('20020-01-01', 'Amazon', 'good')
+  end
```

Comment:
- [optional] It would be great if you also write a test case for the `can_be_archived?` method 🧪

Reply:
write test case for `can_be_archived `